### ISSUE TITLE
Enabling passcode yields blank white screen (AND MORE!) [iPhone 8, iOS 16.1.2]

### ISSUE THREAD

**Issue by dot-screen** [standardnotes/forum#2309]
**Describe the issue**
Enabling pass-code yields a blank white screen.

**To Reproduce**
1. (Pass-code and fingerprint security features have been enabled for ages and working fine.)
2. Uninstall Standard Notes from iPhone.
3. Reinstall Standard Notes. (Now pass-code and fingerprint are disabled upon reinstall.)
4. Enable pass-code security feature.

**[Update, 7 Dec 2022: Tapping on search items now also ends up with the same issue.]**
**To Reproduce:**
1. (Note: Passcode and fingerprint remain disabled.)
2. Tap on the "Search..." bar at the app's header and type a search term.
3. Tap on any search items that appear.

**Expected behavior**
Normal behavior as usual--notes should appear.

**Screenshots**
![standard notes blank 2](https://user-images.githubusercontent.com/100340602/204078790-909d483a-133e-4873-a15c-1663c0ced542.JPG)

**Smartphone (please complete the following information):**
 - Device: iPhone 8
 - OS: iOS 16.1.1
 - Browser: N/A, I use the official iPhone app
 - Version: I can't find the version because of the issue, but I reinstalled the app just now.

Thank you.

**Comment by effieeee** [comment]
Hi @dot-screen, I did actually find a similar report in the help inbox, can you also send us your account email at help@standardnotes.com in case there might be some sort of account related issue?

**Comment by dot-screen** [comment]
Hi @effieeee,

Done and thank you!

**Comment by feinersaft** [comment]
@dot-screen You can also find the app version in the App Store app :)

I too have the same issue on iPhone SE, iOS 16.1.1, App Version 3.114.0

**Comment by dot-screen** [comment]
Thanks, @feinersaft!

I just installed the first update after I posted this issue. The version I have now is 3.114.0

**Comment by BobWs** [comment]
guess this #1761 could be related!

**Comment by homberger** [comment]
Same for me.

**Comment by feinersaft** [comment]
Is there anything we can provide to help speed up the bugfixing? We cannot use this app for almost two weeks due to this bug.

**Comment by dot-screen** [comment]
**More ways to reproduce blank white screen:**
1. (Note: Passcode and fingerprint remain disabled.)
2. Open any note.
3. Edit that opened note (or even not).
4. After editing (or not), tap **either** `Done` **or** the "back" button on the upper left corner, which is supposed to bring you back to your list of notes.

**Smartphone:**
- Device: iPhone 8
- OS: iOS 16.1.2 (updated since this issue was posted)
- Browser: N/A, I use the official iPhone app
- Version: 3.125.0

Thank you.

**Comment by effieeee** [comment]
Fix for white screens has been pushed on [v3.127.2](https://github.com/standardnotes/app/commit/91f3295cffd54c7a5de01633add9b6abdb769cdf), for iOS, update is still pending on the App Store! 🙏

**Comment by homberger** [comment]
And maybe have a look at this comment:
https://github.com/standardnotes/forum/issues/2311#issuecomment-1360851482

The problem was fixed for me by playing with some settings of my iPhone. Without updating the app to the most recent version.

**Comment by dot-screen** [comment]
> Fix for white screens has been pushed on [v3.127.2](https://github.com/standardnotes/app/commit/91f3295cffd54c7a5de01633add9b6abdb769cdf), for iOS, update is still pending on the App Store! 🙏

@effieeee Thank you! Around when will this update roll out?

**Comment by effieeee** [comment]
Hi @dot-screen , unfortunately it looks like the update hasn’t quite corrected the issue, but the temporary workaround in the comment linked above can help while we look into this further

**Comment by dot-screen** [comment]
Just reporting that I just followed the workaround discovered by @homberger and the white screen is now gone for all the 3 instances/ways I identified in this Issue.

Specifically, I only did the following:
Settings -> Accessibility -> Motion -> disabled and re-enabled "Reduce Motion"

Thank you, @homberger!

Smartphone:
Device: iPhone 8
OS: iOS 16.2
Browser: N/A, I use the official iPhone app
Version: 3.125.0 (No newer updates available yet)

@effieeee Thank you. I hope this gets fixed. This amazing app is vital to my daily life!
